### PR TITLE
Add @DesleyAalderink

### PR DESCRIPTION
**Pull request by DesleyAalderink**

### CODE REVIEW

File: site/class-2-style/DesleyAalderink/readme.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+by [**@mbostock**][block-author] (GPL-3.0).
+
+I changed the structure to my liking
+
+[![][cover]][url]
+
+## Background
+
+The assignment was to change the HTML/JS/CSS structure to my own style. For the HTML I always use "" voor linking files. 
+I also closed the <body> tagg. For the CSS, it was pretty much how I always style my CSS. I always use one tab if a code line is
```
Comment (wooorm) [inline site/class-2-style/DesleyAalderink/readme.md]: You should wrap `<body>` in backticks! now it creates an actual body tag!

Comment (DesleyAalderink) [reply site/class-2-style/DesleyAalderink/readme.md]: What do you mean?

Comment (wooorm) [reply site/class-2-style/DesleyAalderink/readme.md]: Use `` `<body>` `` instead!

Comment (DesleyAalderink) [reply site/class-2-style/DesleyAalderink/readme.md]: I just did it in my HTML, but what does it mean?
Tried Googling it, but didn't got any results.

Comment (wooorm) [reply site/class-2-style/DesleyAalderink/readme.md]: You can add random HTML in markdown. So if you add `<script>alert(1)</script>`, it actually alerts (although it doesn’t work on GH).

So, if you add `<body>`, it actually creates an HTML tag.

If you use `` `<body>` ``, it shows the tag as code.

To see what happens, click preview now, and see that the body tag doesn’t show!

Comment (DesleyAalderink) [reply site/class-2-style/DesleyAalderink/readme.md]: ooooh now I feel stupid hahah
I will fix it

Comment (wooorm) [reply site/class-2-style/DesleyAalderink/readme.md]: Don’t feel stupid!

**Comment by wooorm** [comment]
Looks good!

**Pull request merged**